c2corg_ui/static/js/documentediting.js

```diff
@@ -374,6 +389,25 @@ app.DocumentEditingController.prototype.handleMapFeatureChange_ = function(
 
 
 /**
+ * It doesn't associate, it just pushes it into associations array.
+ * @private
+ */
+app.DocumentEditingController.prototype.pushDocToAssociations_ = function() {
```

Comment:
I am not sure if I understand you, but creating the association will not directly be done. You open the wizard to create a new outing, and the route is simply added to the associations list. When you send the request to the API, the outing will be created and then the specified associations (this is not yet supported by the API).